### ISSUE TITLE
There should be enable/disable debugger commands

### ISSUE THREAD

**Issue by krassowski** [jupyterlab/jupyterlab#14463]
### Problem

Currently there is a command to show debugger panel (`debugger:show-panel`), but no command to enable or disable debugger (showing panel does not enable debugger). Instead, there is a custom code buried in the button with bug icon handlers:

https://github.com/jupyterlab/jupyterlab/blob/a4bfc580f8b57f8e5f74f82d0ff5a8dc095d007c/packages/debugger/src/handler.ts#L282-L321

This is a problem for:
- accessibility/usability - there is no way to create a shortcut for toggling debugger
- automation/testing - there is no easy way to enable/disable debugger (the bug icon may or may not be visible depending on screen size and number of notebooks open)

### Proposed Solution

Add `debugger:enable` and `debugger:disable` commands. Rework button to use these commands.

In a sense debugger is special as `debugger:toggle` would correspond to showing/hiding the panel. Enabling/disabling debugger is a more involved than just toggling UI elements as it has overhead on the kernel side and may influence computation performance etc.

### Additional context

Other similar commands:

- Extension manager: `extensionmanager:toggle`
- Inspector: `inspector:toggle`, `inspector:close`, `inspector:open`
- `statusbar:toggle`
- `filebrowser:activate`
